index.js

```diff
@@ -523,6 +546,11 @@
       this === other;
   }
 
+  //  Boolean$prototype$lte :: Boolean ~> Boolean -> Boolean
+  function Boolean$prototype$lte(other) {
+    return typeof other === typeof this && this <= other;
+  }
+
```

Comment:
That's fine. Won't this return `true` for `lte(false, new Boolean(false))`? Is that where we came down on this?
```js
lte(false, new Boolean(false)) // true
lte(0, -0) // true
lte('a', new String('a')) // false
```